src/apiCalls.js

```diff
@@ -0,0 +1,24 @@
+const apiData = {
+  allMovieData() {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => {
+      if(!response.ok) {
+        throw new Error('Refresh browser')
+      } else {
+        return response.json()
+      }
+    })
+  },
+  singleMovieData(id) {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+    .then(response => {
+      if(!response.ok) {
+        throw new Error('Refresh browser')
+      } else {
+        return response.json()
+      }
+    })
+  }
+}
+
+export default apiData;
```

Comment:
These two fetches look relatively similar. The endpoint is the only major difference... maybe we could send the entire URL through as an argument? Then we could have one fetch that can do both!